src/main/java/io/github/bananapuncher714/nbteditor/NBTEditorTestPlugin.java

```diff
@@ -0,0 +1,29 @@
+package io.github.bananapuncher714.nbteditor;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import org.bukkit.Bukkit;
+import org.bukkit.plugin.java.JavaPlugin;
+
+public final class NBTEditorTestPlugin extends JavaPlugin {
+
+    @Override
+    public void onEnable() {
+        try {
+            // TODO Do all test here.
+            throw new RuntimeException("Test close");
```

Comment:
This is just a test example if a test throws an exception the try will catch it and write this error into error.txt and workflow will check if there is error.txt or not if yes build will be canceled and echos the error.